tests/js/server/aql/aql-geo.js

```diff
@@ -28,21 +28,23 @@
 /// @author Copyright 2021, ArangoDB GmbH, Cologne, Germany
 ////////////////////////////////////////////////////////////////////////////////
 
-var disableViewTests = true;
-
-var jsunity = require("jsunity");
-var db = require("@arangodb").db;
-var errors = require("@arangodb").errors;
-var helper = require("@arangodb/aql-helper");
-var getQueryResults = helper.getQueryResults;
-var assertQueryError = helper.assertQueryError;
+const disableViewTests = false;
```

Comment:
Maybe just remove that kludge?

Reply:
I think it can be convenient sometime, it's just test so for me better to keep it